tests/bearlib/aspects/ModuleTest.py

```diff
@@ -18,11 +18,12 @@ def test_module(self):
                 coalib.bearlib.aspects.aspectsModule)
 
     def test__getitem__(self):
+        dict_spelling = coalib.bearlib.aspects.Root.Spelling.DictionarySpelling
         # check a leaf aspect
-        for aspectname in ['aspectsYEAH', 'spelling.aspectsYEAH',
-                           'root.SPELLING.aspectsYEAH']:
-            assert (coalib.bearlib.aspects[aspectname] is
-                    coalib.bearlib.aspects.Root.Spelling.aspectsYEAH)
+        for aspectname in ['DictionarySpelling',
+                           'spelling.DictionarySpelling',
+                           'root.SPELLING.DictionarySpelling']:
+            assert (coalib.bearlib.aspects[aspectname] is dict_spelling)
```

Comment:
remove the round brackets.
I am pretty sure I asked you to do that in the last review.